packages/serverpod/lib/src/server/log_manager/log_manager.dart

```diff
@@ -123,10 +122,44 @@ class LogManager {
 
   /// Logs a query, depending on the session type it will be logged directly
   /// to the database or stored in the temporary cache until the session is
-  /// closed. Call [shouldLogQuery] to check if the entry should be logged
+  /// closed. Call [_shouldLogQuery] to check if the entry should be logged
   /// before calling this method. This method can be called asynchronously.
   @internal
-  Future<void> logQuery(Session session, QueryLogEntry entry) async {
+  Future<void> logQuery(
+    Session session, {
+    required String query,
+    required Duration duration,
+    required int? numRowsAffected,
+    required String? error,
+    required StackTrace stackTrace,
+  }) async {
+    var executionTime = duration.inMicroseconds / (1000 * 1000.0);
```

Comment:
Suggestion: Add a constant for the multiplication done here.

We don't need to do this multiplication over and over again at runtime.